azure-iot-device/azure/iot/device/common/http_transport.py

```diff
@@ -45,8 +47,13 @@ def _create_ssl_context(self):
             ssl_context.load_verify_locations(cadata=self._server_verification_cert)
         else:
             ssl_context.load_default_certs()
-        ssl_context.verify_mode = ssl.CERT_REQUIRED
-        ssl_context.check_hostname = True
+
+        if self._cipher:
+            try:
+                ssl_context.set_ciphers(self._cipher)
+            except ssl.SSLError as e:
+                # TODO: custom error with more detail?
```

Comment:
>TODO [](start = 18, length = 4)

it seems TODO gets flagged in pen testing. So may be we should come up with different term. and use it consistently across